critiquebrainz/data/dump_manager.py

```diff
@@ -189,19 +189,25 @@ def public(location, rotate=False):
 
     # 1. COMBINED
     # Archiving all reviews (any license)
-    REVISION_COMBINED_SQL = "SELECT %s FROM revision JOIN review " \
+    REVISION_COMBINED_SQL = "SELECT revision.id, revision.review_id, timestamp, text " \
```

Comment:
As far as I know consistent order of columns during export and import is important. That's what `get_columns` function handles. If the order is indeed important then that function should be replaced with something that would provide a similar functionality.